Content.Shared/ReagentOnItem/Components/SpaceGlueOnItemComponent.cs

```diff
@@ -0,0 +1,13 @@
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Shared.ReagentOnItem;
+
+[RegisterComponent]
+public sealed partial class SpaceGlueOnItemComponent : ReagentOnItemComponent
```

Comment:
You tag the component class with this attribute...
```suggestion
[RegisterComponent, AutoGenerateComponentPause]
public sealed partial class SpaceGlueOnItemComponent : ReagentOnItemComponent
```